api/serializers/coordinator.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+
+from api.models import Coordinator
+
+
+class MinifiedCoordinatorSerializer(serializers.ModelSerializer[Coordinator]):
+    """Serializer for Coordinator model with only id and full_name fields."""
+
+    full_name = serializers.CharField(source="personal_info.full_name")
+    id = serializers.IntegerField(source="personal_info_id")
```

Comment:
`_id`, not `.id`? Same for teacher

Reply:
it's same :)

Reply:
I mean, `personal_info_id` it's still a field of model, which stores id of related personal_info. 
Usually, when you need only id, it's better to use it as I wrote it, because when you write `related_model.id`, django should hit database, load `related_model` and only then get a field. 
It's not a problem here, because we still use other fields of `personal_info`, just general comment.